### DISCUSSION TITLE
加载3Dtiles后, 如何添加剖面？

### DISCUSSION THREAD

**Discussion by kankantian** [zouyaoji/vue-cesium#485] [Q&A]
按照官网: 
```
<vc-primitive-tileset ref="primitive" :url="url" :clippingPlanes="clippingPlanes" @ready="onReady" />

let clippingPlanes = new Cesium.ClippingPlaneCollection({
    planes: [
      new Cesium.ClippingPlane(
        new Cesium.Cartesian3(0.0, 0.0, -1.0),
        0.0
      ),
    ],
    edgeWidth:  1.0,
  })
```
创建clippingPlanes ，使用clippingPlanes添加到vc-primitive-tileset后并没有显示， 可以给个demo吗

**Comment by zouyaoji** [comment]
我抽时间试试看看再答复

**Reply by kankantian** [reply]
thanks!

**Reply by zouyaoji** [reply]
你好，我这边尝试了。
![image](https://github.com/zouyaoji/vue-cesium/assets/24430116/70a423c7-1e2a-45e8-bc23-11f96122cccc)

如果你要绑定:clippingPlanes="clippingPlanes"的话，需要先定义一个const clippingPlanes = ref(null);  然后在vc-viewer的@ready事件来初始化clippingPlanes 。 需要用到Cesium来初始化的都要这样写。 除非你把Cesium.js的引入放到index.html中。

也可以在vc-primitive-tileset组件的ready事件中这样写：
tileset._clippingPlanes = new Cesium.ClippingPlaneCollection({
          planes: [new Cesium.ClippingPlane(new Cesium.Cartesian3(0.0, 0.0, -1.0), 0.0)],
          edgeWidth: 1.0
        })
